package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "node app.js",
-    "lint": "eslint --ext .js,.json .",
+    "lint": "eslint --ext .js,.json . && cat .bithoundrc | eslint --stdin --stdin-filename=f.json",
```

Comment:
Probably not something to worry about but this is now, AFAIK, not going to work in a Windows environment.